CHANGELOG.md

```diff
@@ -23,6 +23,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     if a `NoSuchKey` response is returned by S3. This can behavior can be
     overridden by passing `{ retries: 0 }` as the `retryOptions` argument.
 
+### Removed
+
+- **CUMULUS-1559**
+  - `logToSharedDestination` has been migrated to the Terraform deployment as `log_api_gateway_to_cloudwatch` and will ONLY apply to egress lambdas.
+  Due to the differences in the Terraform deployment model, we cannot support a global log subscription toggle for a configurable subset of lambdas.
```

Comment:
Are the line breaks in the formatting here intentional?

Reply:
Yes.

Reply:
Did you encounter any issue with them? I basically just figured breaking on the end of a sentence made sense but I haven't looked at how Github shows them. I can make it one block instead.

Reply:
More for readability in the raw markdown.

Reply:
I have no issue with it, just was wondering 👍